desc/vmec.py

```diff
@@ -100,9 +103,19 @@ def load(cls, path, L=-1, M=-1, N=-1, spectral_indexing="fringe"):
         inputs["pressure"] = np.zeros((preset, 2))
         inputs["pressure"][:, 0] = np.arange(0, 2 * preset, 2)
         inputs["pressure"][:, 1] = file.variables["am"][:] * p0
-        inputs["iota"] = np.zeros((preset, 2))
-        inputs["iota"][:, 0] = np.arange(0, 2 * preset, 2)
-        inputs["iota"][:, 1] = file.variables["ai"][:]
+        if profile == "iota":
+            inputs["iota"] = np.zeros((preset, 2))
+            inputs["iota"][:, 0] = np.arange(0, 2 * preset, 2)
+            inputs["iota"][:, 1] = file.variables["ai"][:]
+            inputs["current"] = None
+        if profile == "current":
+            inputs["current"] = np.zeros((preset, 2))
+            inputs["current"][:, 0] = np.arange(0, 2 * preset, 2)
+            inputs["current"][:, 1] = file.variables["ac"][:]
+            inputs["current"][:, 1] *= (
+                file.variables["ctor"][:] / (np.sum(inputs["current"][:, 1]) or 1),
+            )
+            inputs["iota"] = None
```

Comment:
I'm not sure this is correct. The VMEC `ac` output is whatever coefficients were used in the input file, not the actual coefficients of current profile. There are two concerns:
1. The default VMEC input is to give the derivative of the current profile wrt the VMEC radial coordinate s, but DESC expects the full profile as a function of rho. I think we are missing an integration factor here. 
2. The current profile also needs to be scaled to the total value of `ctor`, which you are already accounting for. 

See `VMECIO.parse_vmec_inputs` starting at line 1309 of `input_reader.py` for how both of these are handled already.

Reply:
Think this is fixed now

Reply:
It looks correct now, but have you tested it?

Reply:
Yes I tested it locally, it's basically copy pasted from the input reader so I figured we don't need another test for it (plus that means another saved vmec thing to compare against etc)